pkg/cli/cli.go

```diff
@@ -0,0 +1,107 @@
+package cli
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/cyberark/conjur-oss-suite-release/pkg/changelog"
+	"github.com/cyberark/conjur-oss-suite-release/pkg/github"
+	"github.com/cyberark/conjur-oss-suite-release/pkg/http"
+	"github.com/cyberark/conjur-oss-suite-release/pkg/repositories"
+	"github.com/cyberark/conjur-oss-suite-release/pkg/template"
+)
+
+type Options struct {
+	APIToken           string
+	Date               time.Time
+	OutputFilename     string
+	OutputType         string
+	RepositoryFilename string
+	Version            string
+}
+
+func RunParser(options Options) {
+	templateInfo, err := template.GetTemplateInfo(options.OutputType)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Since this code is moved out of `main.go`, these fatals should change to regular errors for easier testing and the only fatal should happen in `main.go`.